src/main/resources/org/primefaces/Messages_de.properties

```diff
@@ -36,9 +36,9 @@ primefaces.picklist.ADD_ALL = Alle hinzufügen
 primefaces.picklist.REMOVE = Entfernen
 primefaces.picklist.REMOVE_ALL = Alle entfernen
 primefaces.picklist.MOVE_UP = Nach oben verschieben
-primefaces.picklist.MOVE_TOP = Nach oben verschieben
+primefaces.picklist.MOVE_TOP = An den Anfang verschieben
```

Comment:
FYI in Dutch I left out the equivalent of verschieben (as with German, translations can get long compared to English).

Reply:
Your call - But as this is a tooltip there is enough room to be verbose I think